services/supabase/vacancyService.ts

```
@@ -123,66 +127,134 @@ class VacancyService {
-        .range(start, end);
+      // Apply sorting based on the sortBy parameter
+      switch (sortBy) {
+        case 'latest':
+          vacancyLogger.info('Ordering results by posted_date DESC (latest first)');
+          query = query.order('posted_date', { ascending: false });
+          break;
+        case 'expiring':
+          vacancyLogger.info('Ordering results by closing_date ASC (expiring soon first)');
+          query = query.order('closing_date', { ascending: true });
+          break;
+        case 'recommended':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)